docs/fixes/infinite-recursion-policy-fix.md

```
@@ -0,0 +1,70 @@
+# Fix: Infinite Recursion in RLS Policies
+
+## Issue
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)